packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -181,7 +182,7 @@ class RabbitResourceTrackingBaseMessage(RabbitMessageBase):
         ..., description="uniquely identitifies the service run"
     )
     created_at: datetime.datetime = Field(
-        default_factory=lambda: datetime.datetime.now(datetime.timezone.utc),
+        default_factory=lambda: arrow.utcnow(),
```

Comment:
hmm, I do not think that this returns a datetime object.
don't you need to call ```arrow.utcnow().datetime``` ????

Reply:
good catch! thx